sdk/src/testIntegration/java/com/hedera/hashgraph/sdk/test/integration/TokenRejectIntegrationTest.java

```diff
@@ -682,7 +697,7 @@ void canRemoveAllowanceWhenExecutingTokenRejectForFtAndNft() throws Exception {
             .execute(testEnv.client)
             .getReceipt(testEnv.client);
 
-        // verify the allowance - should be 0 , because the receiver is no longer the owner
+        // verify that spender doesn't have the allowance
```

Comment:
isn't the behaviour the same as with FTs - keeping the allowances?

Reply:
Nope, error is different as I tested.